### PR TITLE
Add Grammarly recipe

### PR DESCRIPTION
**Pull request by DavidUser**

### CODE REVIEW

File: recipes/grammarly/package.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "grammarly",
+  "name": "Grammarly",
+  "version": "0.1.0",
```
Comment (vraravam) [inline recipes/grammarly/package.json:4]: Please start with version 1.0.0

File: recipes/grammarly/package.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  "license": "MIT",
+  "config": {
+    "serviceURL": "https://app.grammarly.com/docs/new"
```
Comment (vraravam) [inline recipes/grammarly/package.json:7]: Please confirm that this url is correct. Since I do not have a grammarly account, I'm unable to verify this.

**Review by vraravam — CHANGES_REQUESTED** [review]

**Comment by vraravam** [comment]
Thanks for your contribution @DavidUser !

**Comment by SpecialAro** [comment]
@DavidUser took the liberty of making changes to this PR. If you don't agree that the URL should point to the dashboard (instead of a new document) please let me know.

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @DavidUser for code

**Pull request merged**